### DISCUSSION TITLE
What is Trackshift?

### DISCUSSION THREAD

**Discussion by g105b** [SuperHyperInstantFutureTime/TrackShift#17] [General]
When I asked @richardbirkin, he said "a music royalties data aggregator", which is perfectly accurate, but the choice of words might be a turn off for the target user. Let's discuss a tagline here.

**Comment by richardbirkin** [comment]
You're right. In purely functional terms that's what it is, but it's not what the user wants to do. 

Our first assumption is that the user wants a way to understand and optimise their music earnings. whether that's a DIY artist who gets money from five or six different sources, or a label who managing earnings from five or six sources for multiple artists. it all gets complicated very quickly - what TrackShift does is simplify it.

**Comment by richardbirkin** [comment]
### Notes from 2020

![IMG_5839](https://user-images.githubusercontent.com/115098450/200307706-1acfed44-cb55-497d-b7d2-90bbe6a54d9a.jpeg)
![IMG_5832](https://user-images.githubusercontent.com/115098450/200307657-b235f3c6-f005-4104-a5a9-1e97cf294183.jpeg)
![IMG_5833](https://user-images.githubusercontent.com/115098450/200307666-8207c7b0-fb55-47da-b49e-17c1f1606007.jpeg)
![IMG_5838](https://user-images.githubusercontent.com/115098450/200307701-c4b4cb2f-a46e-4e9d-abac-3b5527ed8344.jpeg)
![IMG_5834](https://user-images.githubusercontent.com/115098450/200307674-e7c6fafb-e2b7-46ad-a161-398afba5cdb0.jpeg)
![IMG_5835](https://user-images.githubusercontent.com/115098450/200307682-d5c48046-be7b-4600-a114-c8f5ba241467.jpeg)
![IMG_5836](https://user-images.githubusercontent.com/115098450/200307691-f2108fed-99d2-452e-b11e-8b0fde7ab471.jpeg)
![IMG_5837](https://user-images.githubusercontent.com/115098450/200307697-74c655a5-92a5-4f1a-9615-92c7d349c203.jpeg)

**Reply by g105b** [reply]
Wow that's a flashback.
